spec/workers/link_check_worker_spec.rb

```diff
@@ -95,14 +97,16 @@
           end
         end
 
-        context 'exceptions' do
+        context 'when there is an exception' do
           it 'handles throttling error' do
             allow(worker).to receive(:link_check).and_raise(ThrottleLimitError.new('ThrottleLimitError'))
+
             expect { worker.perform(link_check_job.id.to_s) }.to_not raise_exception
           end
 
           it 'should handle networking errors' do
```

Comment:
Same here `it 'swallows RestClient Exception'`

Reply:
Done!